### PR TITLE
exporting containers.go to package

### PR DESCRIPTION
**Pull request by mtcherni95**

### CODE REVIEW

**Comment by mtcherni95** [comment]
Solves: #1267

File: pkg/containers/containers.go [hunk 1]
```diff
@@ -12,8 +13,6 @@ import (
 	"syscall"
 	"time"
 	"unsafe"
-
-	bpf "github.com/aquasecurity/libbpfgo"
```
Comment (yanivagman) [inline pkg/containers/containers.go:16]: Why removing it from here?

File: pkg/containers/containers.go [hunk 2]
```diff
@@ -271,7 +270,7 @@ const (
 	containerStarted
 )
 
-func (c *Containers) PopulateBpfMap(bpfModule *bpf.Module) error {
+func (c *Containers) PopulateBpfMap(bpfModule *libbpfgo.Module) error {
 	containersMap, err := bpfModule.GetMap("containers_map")
```
Comment (yanivagman) [inline pkg/containers/containers.go:274]: containers_map shouldn't be hard-coded if we move this to a package.
Let's change this to get the map name as an additional arugment

File: pkg/containers/containers.go [hunk 3]
```diff
@@ -287,7 +286,7 @@ func (c *Containers) PopulateBpfMap(bpfModule *bpf.Module) error {
 	return err
 }
 
-func RemoveFromBpfMap(bpfModule *bpf.Module, cgroupId uint64) error {
+func RemoveFromBpfMap(bpfModule *libbpfgo.Module, cgroupId uint64) error {
 	containersMap, err := bpfModule.GetMap("containers_map")
```
Comment (yanivagman) [inline pkg/containers/containers.go:290]: same

File: pkg/containers/containers.go (hunk 2 above)
Comment (mtcherni95) [reply pkg/containers/containers.go:275]: Agree

**Review by yanivagman — APPROVED** [review]

**Pull request merged**